### DISCUSSION TITLE
Control+Left/Right should navigate by word, not browser history

### DISCUSSION THREAD

**Discussion by aspiers** [rotki/rotki#8982] [General]
It is a universal convention spanning multiple decades of human/computer interface design (at least on Windows and Linux, not sure about MacOS) that when editing an input field, pressing Control+Left will jump back one word.  However in Rotki (at least with v1.36.1 AppImage on Linux) it seems that Control+Left will always navigate directly back in the browser history.  Similarly Control+Right navigates forwards in the browser history instead of adhering to the convention of jumping forward by a word.

So for example if I'm in the middle of filling out the form to add a new account, and I press Control+Left thinking I can jump backwards a word in the current text input field, it exits the entire page and all the values I filled out are lost, with no way to get them back.  So then I have to re-navigate to the page, and fill out the whole form again, suppressing the natural instinct honed over multiple decades of pressing Control+Left when I want to navigate by word.  This is not a hypothetical scenario but actually happened to me ~_twice_~ _three times_ in a row just now - the habit is so engrained that even after figuring out I should avoid that shortcut I still pressed it!

Filing as a discussion rather than an issue in case this is somehow controversial, but IMHO it's definitely a UX bug because it violates the Principle of Least Surprise.

**Comment by LefterisJP** [comment]
Interesting. Definitely would say if possible we should fix this. I don't use ctlr + arrows in rotki for some reason though I am awayre of the combo. So had not noticed this is not respected and is back / forward. We should fix this.

**Reply by aspiers** [reply]
Thanks.  Would you like me to create an actionable issue for this?  Can be done very easily:

![image](https://github.com/user-attachments/assets/a1fc0c7a-ef81-4d06-96f0-b0d9bca17ec0)

**Reply by aspiers** [reply]
I also noticed that Alt+Left/Right (sometimes a.k.a. Meta+Left/Right) don't do anything, even though those are the most conventional keyboard shortcuts for navigating back/forward in browser history.  So probably the most ideal fix would be to change these shortcuts from Control+Left/Right to Alt+Left/Right.

**Comment by lukicenturi** [comment]
Should be fixed by https://github.com/rotki/rotki/pull/8995
